mediaqueries-5/Overview.bs

```diff
@@ -148,6 +148,52 @@ Detecting the ambient light level: the 'light-level' feature</h3>
 		</pre>
 	</div>
 
+<h3 id="reality">
+Detecting the display technology: the 'reality' feature</h3>
+
+<pre class='descdef mq'>
+Name: reality
+Value: 2D | mixed | virtual
```

Comment:
None of these values are falsy. I suspect you might want `mixed` and `virtual` to be truthy, and `2d` to be falsy, so maybe something like:
```
vr: none | mixed | full
```

Reply:
I think it depends.
I don't think "full" is a good term since VR is limited compared to AR

Reply:
Over in the WebXR group, when we migrated from WebVR, we went with XR vs MR or AR or <select your favorite acronym> because these terms have been muddled up by competing marketing departments.  "Mixed Reality" is used in different ways by different companies (eg.  Microsoft, MagicLeap, Meta, etc use AR, VR and MR to mean different things).  We settled on XR because it was the only term left that was all-encompassing of the "various kinds of reality".   So, can I suggest "xr:" as the property name, since it's going to be the name of that standard, and javascript devs will access it through navigator.xr?